server-utils.js

```diff
@@ -86,11 +86,13 @@ module.exports = {
     }
   },
 
-  /**
-   * Only to be used when handling unexpected errors.
-   */
   serverError: function(err, req, res) {
     console.error('Server error:\n', err);
-    respond(req, res, 500, 'Server error');
+    if (err.expected) {
+      const status = err.status || 500;
+      respond(req, res, status, err.message);
+    } else {
+      respond(req, res, 500, 'Server error');
+    }
```

Comment:
OK I'm going to revert any of the error stuff that's been done (FWIW they were in the original prototype), as that's the easiest way to get this 🚋  moving.

I think we want to look at cleaning up this stuff at some point though, it's unhelpful that the default failure in api is to throw a non-json string out. Unless I'm misunderstanding how to use it.